recipes/conversation/conversation.js

```diff
@@ -33,6 +33,44 @@ var tjConfig = {
     }
 };
 
+/*
+//Other configuration parameters you can set include
+var tjConfig = {
+    log: {
+        level: 'info' // valid levels are 'error', 'warn', 'info', 'verbose', 'debug', 'silly'
+    },
+    robot: {
+        gender: 'male', // see TJBot.prototype.genders
+        name: 'Watson'
+    },
+    listen: {
+        microphoneDeviceId: "plughw:1,0", // plugged-in USB card 1, device 0; see arecord -l for a list of recording devices
+        inactivityTimeout: -1, // -1 to never timeout or break the connection. Set this to a value in seconds e.g 120 to end connection after 120 seconds of silence
+        language: 'en-US' // see TJBot.prototype.languages.listen
+    },
+    wave: {
+        servoPin: 7 // corresponds to BCM 7 / physical PIN 26
+    },
+    speak: {
+        language: 'en-US', // see TJBot.prototype.languages.speak
+        voice: undefined, // use a specific voice; if undefined, a voice is chosen based on robot.gender and speak.language
+        speakerDeviceId: "plughw:0,0" // plugged-in USB card 1, device 0; see aplay -l for a list of playback devices
+    },
+    see: {
+        confidenceThreshold: {
+            object: 0.5,   // only list image tags with confidence > 0.5
+            text: 0.1     // only list text tags with confidence > 0.5
+        },
+        camera: {
+            height: 720,
+            width: 960,
+            verticalFlip: false, // flips the image vertically, may need to set to 'true' if the camera is installed upside-down
+            horizontalFlip: false // flips the image horizontally, should not need to be overridden
+        }
+    }
+};
+ */
+
```

Comment:
Not a huge fan of leaving this huge commented block inside the code.  It might be better to put it in the README where you can explain, "here are other ways to configure tjbot" and then link to this section in tjbotlib.  Otherwise, you run the risk of a) having to repeat this block across all 3 recipes, and b) keeping track of this and updating it any time we make a change to tjbot's config.